### DISCUSSION TITLE
No print bed showing

### DISCUSSION THREAD

**Discussion by gav3d** [OrcaSlicer/OrcaSlicer#3284] [Q&A] [answered]
I've just installed on my Linux pc, set up my printer Ender 3 Pro running Klipper, and it doesn't show the print bed on the Prepare  screen. it's just white. When i add a new project, it doesn't show its just white screen but does show in recent projects. Can anyone help?

**Comment by Malesenka** [comment]
-

**Reply by anh977** [reply, marked as answer]
Same problem.  Everything seems to work just nothing is visible in the printbed window.  I can import stls, save projects, even slice the plate. I'm running Ubuntu 20.4 1.9.0, 1.9.1 both do the same thing.  Anything below those versions locks up with a dark "untitled window" and nothing works.

**Reply by gav3d** [reply]
I fount out the problem is Nvidia graphics is coursing the problem. I had an Nvidia graphics card. I switched to onboard Intel graphics and everything shows and works as it should. Strange but it worked

**Reply by nattymcfly5** [reply]
Dealing with this now on v2.2.0
